doc/source/testing.rst

```diff
@@ -33,6 +33,150 @@ Like many packages, MDAnalysis uses `pytest
+
+.. _continuous-integration:
+
+Continuous Integration tools
+============================
+
+When you submit your pull request, several continuous integration tools will run a suite of tests. These should all pass before your code can be merged into MDAnalysis. You can check tests locally by :ref:`running the test suite <run-test-suite>`.
+
+If your pull request fails immediately with an :ref:`appveyor` error, it is likely that you have merge conflicts with the latest code in the ``develop`` branch. :ref:`Rebase your code <rebase-code>` and update your branch by pushing your changes.
+
+If you get an error with :ref:`travis`, it is likely that you've failed a particular test. You should update your code and push again.
+
+If you get :ref:`codecov` errors, this means that your changes have not been adequately tested. Add new tests that address the "missed" lines, and push again.
+
+Ideally, you want all tests to pass. This will look like:
+
+    .. image:: images/ci_checks_passed.png
+
+.. _appveyor:
+
+--------
+Appveyor
+--------
+
+`AppVeyor`_ is a continuous integration/continuous deployment service. MDAnalysis uses it for `testing builds on Windows`_.
+
+Builds are configured in the file ``.appveyor.yml``. If you add a new dependency to MDAnalysis, you will need to add it to the ``$CONDA_DEPENDENCIES`` or ``$PIP_DEPENDENCIES`` in ``.appveyor.yml`` to pass tests.
+
+.. _`testing builds on Windows`: https://ci.appveyor.com/project/orbeckst/mdanalysis
+
+
+.. _travis:
+
+------
+Travis
+------
+
+`Travis is a continuous integration service <https://travis-ci.com/MDAnalysis/mdanalysis>`_ for Linux and MacOS. MDAnalysis uses it for exhaustive testing on Linux systems, and some testing on MacOS. If you add a new dependency to MDAnalysis, you will need to add it to the ``$CONDA_DEPENDENCIES`` or ``$PIP_DEPENDENCIES`` in ``.travis.yml`` to pass tests.
+
+
+.. _codecov:
+
+-------
+Codecov
+-------
+
+Code coverage measures how many lines, and which lines, of code are executed by a test suite. Codecov is a service that turns code coverage reports into a single visual report. Each line is described as one of three categories:
+
+    - a **hit** indicates that the source code was executed by the test suite.
+    - a **partial** indicates that the source code was not fully executed by the test suite; there are remaining branches that were not executed.
+    - a **miss** indicates that the source code was not executed by the test suite.
+
+Coverage is the ratio of ``hits / (sum of hit + partial + miss)``. See the `Codecov documentation <https://docs.codecov.io/docs/about-code-coverage>`_ for more information.
+
+MDAnalysis aims for 90% code coverage; your pull request will fail the Codecov check if the coverage falls below 85%. You can increase coverage by writing futher tests.
```

Comment:
Higher up in the file you had "We strive for >85% code coverage" which contradicts the statement here. Please bump up the earlier statement to ≥90% as well.